build.sbt

```diff
@@ -1,26 +1,34 @@
 
 lazy val commonSettings = Seq(
   scalaVersion := "2.11.8",
-  addCompilerPlugin("org.spire-math" %% "kind-projector" % "0.8.0")
+  addCompilerPlugin("org.spire-math" %% "kind-projector" % "0.8.0"),
+  libraryDependencies += "org.scala-lang" %  "scala-reflect" % scalaVersion.value
 )
 
 lazy val core = project
   .in(file("modules/core"))
   .settings(commonSettings)
   .settings(
     libraryDependencies ++= Seq(
-      "org.tpolecat" %% "doobie-core"               % "0.3.0-M1", // uh, lame
-      "org.tpolecat" %% "doobie-contrib-postgresql" % "0.3.0-M1"
+      "org.scalaz"  %% "scalaz-core" % "7.2.4",
+      "com.chuusai" %% "shapeless"   % "2.3.1"
```

Comment:
We're experimenting with self-describing data types so we can build a generic sequence editor, and I think shapeless will help eliminate some boilerplate.